### PR TITLE
Update default-sharepoint-groups.md

### PR DESCRIPTION
**Pull request by melitoSP**
Updated to reflect that there is no blocking of users adding Everyone but external users on group-connected team sites.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/default-sharepoint-groups.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ The following table describes the SharePoint groups that are created when you us
...
+ **Everyone except external users** All users added to your organization automatically become members of "Everyone except external users". Please note that you cannot change default permissions granted to "Everyone except external users" on Microsoft 365 group-connected team sites. If a group-connected team site is set to "Public," "Everyone except external users" has a default permission level of "Edit." When a group-connected team site is set to "Private," "Everyone except external users" can be manually added to the site through "Site permissions". To change the privacy setting for a group-connected team site, select the Settings icon, and then select **Site information**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/default-sharepoint-groups.md:88]:
```suggestion
 **Everyone except external users** All users added to your organization automatically become members of "Everyone except external users". Please note that you cannot change default permissions granted to "Everyone except external users" on Microsoft 365 group-connected team sites. If a group-connected team site is set to "Public", "Everyone except external users" has a default permission level of "Edit". When a group-connected team site is set to "Private", "Everyone except external users" can be manually added to the site through "Site permissions". To change the privacy setting for a group-connected team site, select the Settings icon, and then select **Site information**.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @melitoSP, kindly commit my suggested changes to correct the punctuation errors. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Holland-ODSP** [comment]
Good morning @scanum , can you help @melitoSP and I get this PR merged? Looks like the edits were approved a few weeks ago.

**Comment by Holland-ODSP** [comment]
Closing this ticker. I opened a new PR in the private repo. #3717

**Pull request closed without merging**